docs/community/bindings.md

```diff
@@ -48,3 +48,7 @@ As a C library, bindings can be made to call H3 functions from different program
 ## Ruby
 
 - [StuartApp/h3_ruby](https://github.com/StuartApp/h3_ruby)
+
+## ECL
```

Comment:
minor request, would you mind reordering ECL between C# and Erlang, to keep alphabetical order?

Reply:
...done